scss/package.json

```diff
@@ -29,8 +29,9 @@
     "node-sass": "^6.0.1",
     "postcss-import": "^14.0.2",
     "rollup": "^2.56.0",
-    "rollup-plugin-postcss": "^4.0.0",
     "rollup-plugin-scss": "^3.0.0",
-    "rollup-plugin-visualizer": "^5.5.2"
+    "rollup-plugin-visualizer": "^5.5.2",
```

Comment:
Let's drop this. I think it only works for JS, and since this is SCSS only, it's not going to output anything for us. Remove any reference to it in the rollup config file, too.